examples/compiled/area.vg.json

```diff
@@ -83,7 +83,6 @@
       "scale": "x",
       "orient": "bottom",
       "grid": true,
-      "tickCount": {"signal": "ceil(width/40)"},
```

Comment:
Why should this affect the output Vega at all?

Reply:
Ahh, it looks like the difference is that I didn't add tickCount to the grid axis.

Reply:
Fixed.